libclc/generic/libspirv/atomic/atomic_load.cl

```diff
@@ -31,8 +31,9 @@ TYPE __clc__atomic_##PREFIX##load_##AS##_##BYTE_SIZE##_##MEM_ORDER(volatile AS c
   }
 
 #define IMPL_AS(TYPE, TYPE_MANGLED, PREFIX, BYTE_SIZE) \
```

Comment:
Are you able to format this whole expression while you're here? It'd be nice to slowly improve the formatting of libclc as we go.

Reply:
Done :+1: